go-controller/pkg/cluster/gateway_init_linux_test.go

```diff
@@ -295,8 +295,10 @@ var _ = Describe("Gateway Init Operations", func() {
 				"ip link set br-nexthop up",
 				"ip addr flush dev br-nexthop",
 				"ip addr add 169.254.33.1/24 dev br-nexthop",
+				"ip route flush table 6",
```

Comment:
I'm not sure we should do this; restarting ovn-kubernetes should break traffic. Instead, we should properly sync. That means removing stale IPs and adding missing ones on restart.